.gitignore

```diff
@@ -1,5 +1,12 @@
 # secrets
 credentials.properties
+app/src/main/cpp/secrets.cpp
+app/src/main/cpp/secrets.hpp
+app/src/main/cpp/sha256.cpp
+app/src/main/cpp/sha256.hpp
+app/src/main/cpp/CMakeLists.txt
+app/src/main/java/org/greatfire/wikiunblocked/Secrets.kt
+app/.cxx
```

Comment:
in fdroid we just have the cpp folder and the secrets file.  i don't think the .cxx folder is related

Reply:
it looks like it might be intermediate build files.  i don't see that directory in the current repo though so i guess also ignoring it is fine